storage/feed.go

```diff
@@ -195,23 +195,23 @@ type StoreEventListener interface {
 // will run until the Subscription's events channel is closed.
 func ProcessStoreEvents(l StoreEventListener, sub *util.Subscription) {
 	for event := range sub.Events() {
-		switch event := event.(type) {
+		switch specificEvent := event.(type) {
```

Comment:
Shadowing in a type switch is idiomatic go; I'd rather fix go-nyet for this than have to use a cumbersome name like `specificFoo` wherever we use this pattern.